### PR TITLE
Reinstate test3b

### PR DESCRIPTION
**Pull request by bbuzz31**
This reinstates the ray tracing and slant projection test, comparing a single location to a golden value and using the already staged weather files.

It also exposes the maximum tropo height that is used for the ray tracing integral, and sets the new default to 80 km (used to be 50 km).

Part of #535 
Replaces #548

### CODE REVIEW

File: test/scenario_2/stations2.csv
Comment (bbuzz31) [inline test/scenario_2/stations2.csv:1]: this is for the GNSS test; the coordinates are import to match the existing wm file

File: tools/RAiDER/models/ecmwf.py
Comment (bbuzz31) [inline tools/RAiDER/models/ecmwf.py:1]: https://climatedataguide.ucar.edu/climate-data/era5-atmospheric-reanalysis

File: tools/RAiDER/models/weatherModel.py [hunk 1]
```diff
@@ -202,12 +202,11 @@ def set_latlon_bounds(self, ll_bounds, Nextra=2):
...
         ex_buffer_lon_max = 0.0
 
-        if self._Name == 'GMAO' or self._Name == 'MERRA2':
-            ex_buffer_lon_max = self._lon_res
-
-        elif self._Name in 'HRRR HRRR-AK HRES'.split():
+        if self._Name in 'HRRR HRRR-AK HRES'.split():
             Nextra = 6 # have a bigger buffer
 
+        else:
```
Comment (bbuzz31) [inline tools/RAiDER/models/weatherModel.py:208]: for all models buffer by at least one grid cell

File: tools/RAiDER/models/weatherModel.py [hunk 2]
```diff
@@ -217,11 +216,14 @@ def set_latlon_bounds(self, ll_bounds, Nextra=2):
 
         S= np.max([S - pixlat, -90.0 + pixlat])
         N= np.min([N + pixlat, 90.0 - pixlat])
-        W= np.max([W - pixlon, -180.0 + pixlon])
+        W= np.max([W - (pixlon + ex_buffer_lon_max), -180.0 + (pixlon+ex_buffer_lon_max)])
```
Comment (bbuzz31) [inline tools/RAiDER/models/weatherModel.py:219]: buffer both sides by on the lons

**Review by jlmaurer — APPROVED** [review]
LGTM

**Pull request merged**